### ISSUE TITLE
How to make nn.HybridSequential handel multiple input/output layer?(Gluon)

### ISSUE THREAD

**Issue by PistonY** [apache/incubator-mxnet#14762]
For example I have a custom layer defind as
```python
class my_layer(nn.HybridBlock):
    def __init__(self, **kwargs):
        super(my_layer, self).__init__(**kwargs)
        self.conv1 = nn.Conv2D(32, 1, 1)
        self.conv2 = nn.Conv2D(32, 1, 1)
    
    def hybrid_forward(self, F, x1, x2):
        y1 = self.conv1(x1)
        y2 = self.conv2(x2)
        return y1, y2
```
And if use `nn.HybridSequential`
```python
seq = nn.HybridSequential(my_layer(), my_layer())
```
Then make data and run would raise an error
```
HybridBlock requires the first argument to forward be either Symbol or NDArray, but got <class 'tuple'>
```

Do you have any way to fix this?

**Comment by frankfliu** [comment]
@mxnet-label-bot add [gluon, question]

**Comment by lanking520** [comment]
@PistonY The way you are using is incorrect. You should follow this:
https://discuss.mxnet.io/t/gluon-multi-input-block/1993

```
net = gluon.nn.HybridSequential()
net.add(my_layer())
net.add(my_layer())
```

**Comment by PistonY** [comment]
Sorry.That‘s not work to me.I have two different size input/output which can’t concat to one tensor.

**Comment by PistonY** [comment]
I inherit `HybridSequential` to fix this.
```python
class DP_Sequential(nn.HybridSequential):
    def __init__(self, prefix=None, params=None):
        super(DP_Sequential, self).__init__(prefix=prefix, params=params)

    def hybrid_forward(self, F, x1, x2):
        for block in self._children.values():
            x1, x2 = block(x1, x2)
        return x1, x2
```

**Comment by RuRo** [comment]
It's really bizarre, that `gluon` doesn't support this out of the box. Returning a "`tuple of NDArray/Symbol`" is such a common use case and it is even supported in some other `mxnet` APIs (i.e. datasets/dataloaders).
